setup/depends.sh

```diff
@@ -17,69 +17,64 @@
 # #########################
 #     value
 # #########################
-pgram=brew
+current_path="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-if [ "$(id -u)" != "0" ]; then
-    echo "[!] This script must be run as root." 1>&2
-    exit 1
+if [[ "$EUID" != "0" ]]; then
```

Comment:
Checking the EUID should be sufficient here. If you're okay with this change, see the comments further down.